book/07-git-tools/sections/debugging.asc

```diff
@@ -31,18 +31,18 @@ b8b0618cf6fab (Cheng Renquan  2009-05-26 16:03:07 +0800 70)   KBUILD_VERBOSE = $
 066b7ed955808 (Michal Marek   2014-07-04 14:29:30 +0200 82) endif
 ----
 
-Notice that the first field is the partial SHA-1 of the commit that last modified that line.
-The next two fields are values extracted from that commit -- the author name and the authored date of that commit -- so you can easily see who modified that line and when.
-After that come the line number and the content of the file.
-Also note the `^1da177e4c3f4` commit lines, where the `^` prefix designates lines that were introduced in the repository's initial commit and have remained unchanged ever since.
-This is a tad confusing, because now you’ve seen at least three different ways that Git uses the `^` to modify a commit SHA-1, but that is what it means here.
+Pansinin na ang unang patlang ay ang bahagyang SHA-1 ng commit na huling nabago sa linyang iyon.
+Ang sunod na dalawang mga patlang ay mga nakuhang halaga mula sa commit na iyon -- ang pangalan ng may-akda at ang petsa ng pag-akda sa pag-commit na iyon -- kaya madali mong makita kung sino ang nagbago sa linya at kung kailan.
+Pagkatapos ay may kasunod na bilang ng linya at ang nilalaman ng file.
+Tandaan din ang `^1da177e4c3f4` na commit na mga linya, na kung saan ang `^` na prefix ay nagtatakda ng mga linya na pinakilala ang paunang commit sa repositoryo at mayroong hindi nabago kailan man.
```

Comment:
`ever since` could be translated to `magmula noon` or `mula noon`

Reply:
done change using mula noon